### PR TITLE
review and update active health areas

### PR DESCRIPTION
**Pull request by ffinger**
Solves #12

### CODE REVIEW

**Comment by thibautjombart** [comment]
Thanks for this. Could you add the preamble from other reports e.g.
https://github.com/reconhub/report_factories_templates/blob/master/linelist_investigations/report_sources/epicurves_2019-10-12.Rmd#L20-L37

And add yourself as first author and maintainer (if you're okay with it)? You should be first author in any case.

File: linelist_investigations/report_sources/active_health_areas_2019-07-12.Rmd [hunk 1]
```diff
@@ -1,364 +1,385 @@
...
+knitr::opts_chunk$set(echo = TRUE,
+                      collapse = TRUE,
+                      fig.width = 8,
+                      fig.height = 6,
+                      dpi = 150,
+                      warning = FALSE,
+                      message = FALSE)
+```
+
+
```
Comment (thibautjombart) [inline linelist_investigations/report_sources/active_health_areas_2019-07-12.Rmd]: Could you add the preamble from other reports e.g.
https://github.com/reconhub/report_factories_templates/blob/master/linelist_investigations/report_sources/epicurves_2019-10-12.Rmd#L20-L37

And add yourself as first author and maintainer (if you're okay with it)? You should be first author in any case.

**Comment by thibautjombart** [comment]
Thanks! Minor stuff. For further PRs, best if you can create branches on the main project repos, though it's fun to juggle with several remotes for testing ;)

**Comment by ffinger** [comment]
Done.
I left the version and data contributors as is and put you as the reviewer.

**Pull request merged**